server/game/cards/03_TWI/events/InPursuit.ts

```diff
@@ -0,0 +1,34 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class InPursuit extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '6401761275',
+            internalName: 'in-pursuit'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Exhaust a friendly unit. If you do, exhaust an enemy unit.',
+            optional: true,
+            targetResolver: {
+                controller: RelativePlayer.Self,
+                cardCondition: (card) => card.isUnit() && !card.exhausted,
```

Comment:
We shouldn't filter out exhausted cards since the player can legally choose to exhaust an already-exhausted friendly card, though it means the `ifYouDo` part will not resolve.

Reply:
removed the filter